CHANGELOG.md

```diff
@@ -1,4 +1,31 @@
-## 3.77.0 (Unreleased)
+## 3.78.0 (Unreleased)
+
+IMPROVEMENTS:
+* composer: added validation for `max_pods_per_node` field. ([#3445](https://github.com/hashicorp/terraform-provider-google-beta/pull/3445))
+* servicenetworking: added support for `user_project_override` and `billing_project ` to `google_service_networking_connection` ([#3455](https://github.com/hashicorp/terraform-provider-google-beta/pull/3455))
+
+BUG FIXES:
+* fixed a crash on `azure_blob_storage_data_source` for `google_storage_transfer_job` ([#3447](https://github.com/hashicorp/terraform-provider-google-beta/pull/3447))
+* bigquerydatatransfer: fixed a bug where `destination_dataset_id` was required, it is now optional. ([#3438](https://github.com/hashicorp/terraform-provider-google-beta/pull/3438))
```

Comment:
```suggestion
* bigquery: fixed a bug in `google_bigquery_data_transfer_config` where `destination_dataset_id` was required, it is now optional . ([#3438](https://github.com/hashicorp/terraform-provider-google-beta/pull/3438))
```

Reply:
I think this one is correct, BQ data transfer is a separate API

Reply:
You're totally right. While confirming what tag we used before, I saw that this note was already a part of the previous changelog entry, so I'm editing my suggestion to remove the line entirely

Reply:
Oh, unusual- I wonder what caused it to show up twice.